test/readMPAS.f90

```diff
@@ -94,6 +113,25 @@ subroutine loadMPASMesh(mpMesh, filename)
                       !i=numMPs leads to mp2Elm(numMPs=nCells+2)=numMPs-2=nCells
     end do
     call polympo_createMPs(mpMesh,nCells,numMPs,c_loc(mpsPerElm),c_loc(mp2Elm),c_loc(isMPActive))
+
+    !set mp positions
+    allocate(mpPosition(nDims,numMPs))
+    do i = 1,numMPs
+      mpPosition(1,i) = i+ptOne
+      mpPosition(2,i) = numMPs+i+ptOne
+      mpPosition(3,i) = (2*numMPs)+i+ptOne
+    end do
+
+    call polympo_setMPPositions(mpMesh,nDims,numMPs,c_loc(mpPosition))
+    mpPosition = 42
+    call polympo_getMPPositions(mpMesh,nDims,numMPs,c_loc(mpPosition))
+    do i = 1,numMPs
+      if(isMPActive(i) .eq. 1) then
```

Comment:
Please define a parameter with value 1 and use it, e.g., if(isMPActive(i) .eq. MP_ACTIVE)

Reply:
fixed with https://github.com/SCOREC/polyMPO/pull/26/commits/3ad1bfe776e24416282bbd4a16bc8a5cb4503ec2

Reply:
Need: `integer, parameter :: MP_ACTIVE = 1`

Reply:
Thank you.  fixed typo with https://github.com/SCOREC/polyMPO/pull/26/commits/ced82a49c5e5cfd017c954fd3aae8628a52a4679